### PR TITLE
Feature/dynamic max claim rounds

### LINKED ISSUE #233: Fetch maxEarningsClaimsRounds value from BondingManager
**What's the problem? (required)**

The explorer hardcodes 20 as the max number of rounds that a delegator can claim earnings for in a single transaction. Instead, the explorer should fetch the `maxEarningsClaimsRounds` parameter from the BondingManager contract so that even if the parameter value changes, the explorer will still be able to craft transactions that use the updated parameter value as the max number of rounds that a delegator can claim earnings for in a single transaction.

**What's the current behavior? (required)**

The explorer hardcodes 20 as the max number of rounds that a delegator can claim earnings for in a single transaction - see: https://github.com/livepeer/livepeerjs/blob/master/packages/explorer/src/views/ClaimEarnings/index.js#L30

**Describe a possible solution you've considered (optional)**

The SDK already contains a getter function [getMaxEarningsClaimsRounds()](https://github.com/livepeer/livepeerjs/blob/master/packages/sdk/src/index.js#L746).

The [Protocol](https://github.com/livepeer/livepeerjs/blob/master/packages/graphql-sdk/src/fragments/Protocol.js) GraphQL query could be updated to contain the output of this getter function in the result. The query would then be used in the explorer to retrieve `maxEarningsClaimsRounds`.

### PR DESCRIPTION
**Pull request by Randy1Burrell**
**What does this pull request do? Explain your changes. (required)**

Removes the hard coded value 20 that isdicated maxEarningsClaimsRounds and replaced them with values attained from the protocol smart contract. 

**Specific updates (required)**

- Replaces hard coded 20 with maxEarningsClaimsRounds 

**How did you test each of these updates (required)**

I tested manually on Rinkeby by running the explorer and tried to bond from an address that has a high claim.

**Does this pull request close any open issues?**

Fixes #233 

**Screenshots (optional):**

image

**Checklist:**

- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .vimsess [hunk 1]
```diff
@@ -0,0 +1,1404 @@
+let SessionLoad = 1
```
Comment (yondonfu) [inline .vimsess]: Doesn't seem like this file should be committed?

Comment (Randy1Burrell) [reply .vimsess]: Whoops, good catch. Sorry about that man, I thought there was an ignore rule for this. Removing it now.

Comment (Randy1Burrell) [reply .vimsess]: @yondonfu I just removed it.

**Comment by Randy1Burrell** [comment]
@yondonfu can you review this PR for me, please?

**Review by ya7ya — APPROVED** [review]

**Pull request merged**